cpp/ql/src/codeql-suites/cpp-security-extended.qls

```diff
@@ -3,3 +3,6 @@
 - apply: security-extended-selectors.yml
   from: codeql/suite-helpers
 - apply: codeql-suites/exclude-slow-queries.yml
+- exclude:
+    id: cpp/overrun-write
```

Comment:
Noting for completeness that this query still remains in `cpp-security-and-quality.qls`.

Reply:
Do we want to exclude it there to? If so, it might be better to lower the precision instead.

Reply:
Lower the precision instead to make sure we exclude it everywhere.

Reply:
I think this is better, as we've seen this query causing 6hr timeouts, where it seems to hang in the range analysis library that is being used.

Reply:
Reasonable. This will mean that existing alerts previously produced by this query get closed; but arguably that is better than the entire analysis timing out.

Reply:
2.13.4 is the first release in which we would ship this, so I don't think that's a problem.

Reply:
Ah so it wasn't shipped in 2.13.3. Then you are all good.

Reply:
Correct.